shell.nix

```diff
@@ -13,7 +13,7 @@
     nodePackages.typescript-language-server
     nodePackages.npm
 
-    go_1_19
+    go_1_18
```

Comment:
This reverts the go upgrade?

Reply:
No, I was testing out with go 1.19 but then I just removed nix from the integration tests since it was not needed and then reverted this file as it is on develop